client/src/components/J40Header.tsx

```diff
@@ -89,67 +86,36 @@ const J40Header = () => {
           className={'j40-header'}>{intl.formatMessage(messages.timeline)}</Link>],
     ]);
 
-    // select which items from the above map to show, right now it's only two
-    // possibilities so it's simple. Note: strings are used as react keys
-    const menu =
-      ('sprint3' in flags!) ?
-        ['about', 'cejst', 'methodology', 'contact'] :
-        ['about', 'cejst', 'methodology', 'contact'];
-    // TODO: make feature flags flags work.
+    const menu =['about', 'cejst', 'methodology', 'contact'];
     return menu.map((key) => menuData.get(key));
   };
 
   return (
     <>
-      <Helmet htmlAttributes={{lang: intl.locale}}>
-        <meta charSet="utf-8"/>
-        <title>{title}</title>
-      </Helmet>
       <GovBanner/>
       <Header
         basic={true} role={'banner'}
         className={'usa-header j40-header'}>
         <div className="usa-nav-container">
           <div className="usa-navbar">
-            <Title>{title}
-              <div className={'byline'}>
-                A climate and economic justice screening tool
-              </div>
-            </Title>
-
+            <h1 className="usa-logo">
+              <img className="sitelogo" src={siteLogo} alt={title} />
+              <span className={'usa-logo__text'}>{title}</span>
+            </h1>
             <NavMenuButton
               key={'mobileMenuButton'}
               onClick={toggleMobileNav}
               label="Menu"/>
           </div>
           <PrimaryNav
-            items={headerLinks(flags)}
+            items={headerLinks()}
```

Comment:
am I crazy or was this in your other PR?

Reply:
You're not crazy. That branch got super confusing/messy because I started undoing changes I'd made and then merging with main. So, this branch/PR is just pulling those changes over. 

I considered just deleting that branch, but there were PR comments I wanted to keep.

Maybe it would have been better to stay in that PR (which I set to Draft)?

Reply:
I updated the title and description of this PR/branch so it's hopefully less confusing. Sorry about that.